src/plugins/intel_cpu/src/emitters/snippets/aarch64/jit_kernel_emitter.cpp

```diff
@@ -272,14 +271,31 @@ void jit_kernel_static_emitter::init_data_pointers(const std::vector<XReg>& arg_
         h->ldr(data_ptr_regs[num_params + i],
                ptr(reg_runtime_params, static_cast<int32_t>(GET_OFF(buffer_scratchpad_ptr))));
     }
-    for (size_t i = 0; i < num_params; i++) {
-        if (i < num_inputs) {
-            h->ldr(data_ptr_regs[i],
-                   ptr(reg_runtime_params, static_cast<int32_t>(GET_OFF(src_ptrs) + i * sizeof(void*))));
-        } else {
-            h->ldr(data_ptr_regs[i],
-                   ptr(reg_runtime_params, static_cast<int32_t>(GET_OFF(dst_ptrs) + (i - num_inputs) * sizeof(void*))));
-        }
+
+    size_t i = 0;
```

Comment:
Could you please try to extract a common code to a separate function to reuse it here and in `jit_kernel_dynamic_emitter`?